assets/js/plugins.coffee

```diff
@@ -0,0 +1,39 @@
+if module?
+  global._ = require('lodash')
+  global.Logger = require('./logger.coffee')
+  global.DependencyGraph = require('dependencies-online')
+
+(() ->
+  class PluginAPI
```

Comment:
Well the idea is that we actually will add API methods here, although it doesn't do that currently